src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.model;
+
+public enum Operation {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private String code;
+
+    Operation(String code) {
```

Comment:
access modifier?

Reply:
Modifier 'private' is redundant for enum constructors as i know: